improver_tests/acceptance/test_estimate_samos_coefficients.py

```diff
@@ -138,3 +138,29 @@ def test_estimate_samos_coefficients_sites(tmp_path):
     acc.compare(
         output_path, kgo_path, atol=COMPARE_EMOS_TOLERANCE, rtol=COMPARE_EMOS_TOLERANCE
     )
+
+
+def test_estimate_samos_coefficients_no_gam(tmp_path):
+    """
+    Test estimate-samos-coefficients when no GAM is provided. The CLI should return
+    nothing in this instance.
```

Comment:
```suggestion
    None in this instance.
```
It would also be useful to specify why we would want this / what the outcome of this is.